### DISCUSSION TITLE
Does boost support s3 storage?

### DISCUSSION THREAD

**Discussion by wisarmy** [filecoin-project/boost#1566] [Storage Provider]
I want to use s3 for storage, what should I do？

**Comment by LexLuthr** [comment]
Can you please explain your question further?

**Reply by wisarmy** [reply]
We have modified lotus-miner to store sector information on s3 and support retrieval from s3(unsealed). If we switch to boost, will this affect?

**Reply by LexLuthr** [reply]
Apologies for a very delayed response.
1. Boost will require the sector information in it's database.
2. As for the unsealed copies on s3, it would depend on how they are accessed, Boost asks lotus-miner to provide a reader over the unsealed copy. If this functionality is intact then Boost should be able to serve retrievals.
